portfolio/src/main/webapp/reviews.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="/css/main.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div class="container">
+      <nav>
+        <input type="checkbox" id="nav" class="hidden">
+        <label for="nav" class="nav-btn">
+          <i></i>
+          <i></i>
+          <i></i>
+        </label>
+        <div class="nav-wrapper">
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="about.html">About</a></li>
+          <li><a href="projects.html">Projects</a></li>
+          <li><a href="food.html">Food</a></li>
+          <li><a href="reviews.html">Reviews</a></li>
+        </ul>
+        </div>
+      </nav>
+    </div>
+
+    <div class="parallax-background-image black-background-image">
+      <div class="titlecontainer">
+        <span class="homepageheader1">~Food Reviews~</span>
+        <span class="homepageheader2">Gimme your love, or your roasts xD</span>
+      </div>
+    </div>
+
+    <section class="section section-dark">
+      <button onclick="getReviews()">Gimme some reviews!</button>
+      <div id="reviews-container"></div> 
+
+      <form action="/reviews" method="POST">
+        <p>Give me food reviews!</p>
```

Comment:
Closing HTML elements is required for most tags, it's not for <p> or <br> as I understand it. 
https://google.github.io/styleguide/htmlcssguide.html#Optional_Tags

Reply:
Ugh, I didn't realize the template actually used `<p>` and `<br>` lol